report/renderer/performance-category-renderer.js

```diff
@@ -360,7 +360,7 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
     textEl.textContent = Globals.strings.showRelevantAudits;
 
     const filterChoices = [
-      /** @type {const} */ ({acronym: 'All'}),
+      /** @type {const} */ ({acronym: 'All', id: 'All'}),
```

Comment:
This was necessary because of a type error on L380. Just seemed easier to do this than figure out how to narrow the type down there.